### PR TITLE
Update token.move to support SVG as NFT meta data.

### PR DESCRIPTION
**Pull request by jerome-chen**
### Description
Split MAX_URI_LENGTH to MAX_COLLECTION_URI_LENGTH and MAX_TOKEN_URI_LENGTH to support SVG as NFT meta data.

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4798)

### CODE REVIEW

File: aptos-move/framework/aptos-token/sources/token.move [hunk 1]
```diff
@@ -30,7 +30,10 @@ module aptos_token::token {
 
     const MAX_COLLECTION_NAME_LENGTH: u64 = 128;
     const MAX_NFT_NAME_LENGTH: u64 = 128;
-    const MAX_URI_LENGTH: u64 = 512;
+    const MAX_COLLECTION_URI_LENGTH: u64 = 512;
+
+    // to support SVG as meta data, the token uri limits should be increased.
+    const MAX_TOKEN_URI_LENGTH: u64 = 51200;
```
Comment (areshand) [inline aptos-move/framework/aptos-token/sources/token.move:36]: Store the data in off-chain storage like arweave or stored in the default properties in tokendata?

Comment (jerome-chen) [reply aptos-move/framework/aptos-token/sources/token.move:36]: onchain with RFC2986: https://www.rfc-editor.org/rfc/rfc3986 support data URIs.

Comment (jerome-chen) [reply aptos-move/framework/aptos-token/sources/token.move:36]: it's part of ERC721 standard. 

`    /// @notice A distinct Uniform Resource Identifier (URI) for a given asset.
    /// @dev Throws if `_tokenId` is not a valid NFT. URIs are defined in RFC
    ///  3986. The URI may point to a JSON file that conforms to the "ERC721
    ///  Metadata JSON Schema".
    function tokenURI(uint256 _tokenId) external view returns (string);`

https://eips.ethereum.org/EIPS/eip-721

**Comment by davidiw** [comment]
great idea, but it seems better to use the property map instead of this.

**Pull request closed without merging**